src/Parsers/ASTSelectQuery.cpp

```diff
@@ -37,6 +37,8 @@ ASTPtr ASTSelectQuery::clone() const
     for (const auto & child : children)
         res->children.push_back(child->clone());
 
+    res->has_query_parameters = has_query_parameters;
```

Comment:
for this the same comment as above can be applied

Reply:
addressed it